unit_test/sample_key/openssl.cnf

```diff
@@ -12,7 +12,11 @@ keyUsage = nonRepudiation, digitalSignature, keyEncipherment
 subjectKeyIdentifier = hash
 subjectAltName = otherName:1.3.6.1.4.1.412.274.1;UTF8:ACME:WIDGET:1234567890
 extendedKeyUsage = critical, serverAuth, clientAuth, OCSPSigning
-1.3.6.1.4.1.412.274.6 = ASN1:OID:1.3.6.1.4.1.412.274.2
+1.3.6.1.4.1.412.274.6 = ASN1:SEQUENCE:id_spdm_cert_oids
+[id_spdm_cert_oids]
+field1 = SEQUENCE:id_spdm_cert_oid
+[id_spdm_cert_oid]
+field1 = OID:1.3.6.1.4.1.412.274.2

```

Comment:
This seems good to me.

Reply:
I checked "openssl.cnf" and some of the certificates generated using it. These are the expected modifications and no problem.

Reply:
Thanks! Approve it.